src/test/kotlin/no/nav/familie/ef/iverksett/util/BehandlingstatistikkUtil.kt

```diff
@@ -15,7 +15,7 @@ fun opprettBehandlingsstatistikkDto(behandlingId: UUID, hendelse: Hendelse, fort
                                     personIdent = "aktor",
                                     gjeldendeSaksbehandlerId = "saksbehandler",
                                     eksternFagsakId = "eksternFagsakId",
-                                    hendelseTidspunkt = ZonedDateTime.of(LocalDateTime.now(), ZoneId.of("Europe/Oslo")),
+                                    hendelseTidspunkt = ZonedDateTime.of(LocalDateTime.now(), ZoneId.of("UTC")),
```

Comment:
Skjønte ikke denne endringen i testUtil, er det en test viser at konvertering fra utc til Europe/Oslo håndteres?

Reply:
Ja, mest for å være sikker på at vi "setter UTC" i test-caset - for å verifisere at den blir konvertert til Europe/Oslo

Reply:
👍